### PR TITLE
bc, dc: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

# ToDo

- [x] GNU bc
- [x] GNU dc
- [x] FreeBSD bc

### CODE REVIEW

File: pages/common/bc.md [hunk 1]
```diff
@@ -1,20 +1,29 @@
 # bc
 
 > An arbitrary precision calculator language.
-> More information: <https://manned.org/bc>.
+> Get help or version: bc --help|--version.
```
Comment (navarroaxel) [inline pages/common/bc.md:4]: the page has 6 examples, add `--help` and `--version` as examples at the end.
```suggestion
```

File: pages/osx/bc.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# bc
+
+> An arbitrary precision calculator language.
+> Get help or version: bc --help|--version.
```
Comment (navarroaxel) [inline pages/osx/bc.md:4]: the page has 6 examples, add --help and --version as examples at the end.
```suggestion
```

File: pages/common/dc.md [hunk 3]
```diff
@@ -1,24 +1,29 @@
 # dc
 
 > An arbitrary precision calculator. Uses reverse polish notation (RPN).
+> Get help or version: dc --help|--version.
```
Comment (navarroaxel) [inline pages/common/dc.md:4]: the page has 6 examples, add --help and --version as examples at the end.
```suggestion
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/common/bc.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply pages/common/bc.md:4]: But in general case there is no always enough space for such `--help` and `--version` examples. That's why I moved them to the top to mention them always and not to pollute code samples. :) I don't want miss smth more important due `--help` and `--version` examples and I wanna provide info about how view help for user.

Comment (navarroaxel) [reply pages/common/bc.md:4]: moving `--help/--version` to the command description, it's giving to much importance to how to get the `--version` of the program.

Comment (EmilySeville7cfg) [reply pages/common/bc.md:4]: Yes, that's true. :) But on the other hand there are too many tool versions, for instance `mawk`, `gawk` and others. And user have to know how to ensure what command he/she uses. 🤔

Comment (blueskyson) [reply pages/common/bc.md:4]: I'm with @navarroaxel , the purpose of top lines are brief description of a command. Put example here will break tldr's formatting.

Comment (sbrl) [reply pages/common/bc.md:4]: I would argue that `--help` and `--version` are not suitable for inclusion in a tldr page at all, as they are pretty much a standard convention.

**Review by navarroaxel — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**